src/create_sysroot.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Create a linux sysroot image (tar archive) based on debian
+# This is than used in the build scripts as the --sysroot arguments
+# when compiling linux binaries.
+#
+# See https://xw.is/wiki/Create_Debian_sysroots
+
+set -e
+
+SUITE=stretch
+TARGET_DIR=sysroot_debian_${SUITE}_amd64
```

Comment:
if we plan to do more than one version of the sysroot, maybe it should get a number too?

Reply:
We an add that later if needed I think.